pages/common/openscad.md

```diff
@@ -0,0 +1,16 @@
+# openscad
+
+> Software for creating solid 3D CAD objects.
+> More information: <https://openscad.org>.
+
+- Open a file:
+
+`openscad {{button.scad}}`
+
+- Convert a file to STL:
+
+`openscad -o {{button.stl}} {{button.scad}}`
```

Comment:
```suggestion
`openscad -o {{path/to/button.stl}} {{path/to/button.scad}}`
```

Reply:
Does this have long flags E.G. -o to --output

Reply:
@CleanMachine1 It does not.